app/src/main/res/values/styles.xml

```diff
@@ -39,4 +39,31 @@
     <style name="PopupMenu" parent="@style/Widget.AppCompat.Light.PopupMenu">
         <item name="overlapAnchor">true</item>
     </style>
+
+    <style name="AppTheme.ContextMenuItem">
+        <!-- All these attributes bellow need to be defined in a theme using android:theme
+            otherwise they will have no effect... -->
+        <item name="android:listPreferredItemHeightSmall">48dp</item>
+        <!-- Set the padding to the children (menu item) of the NavigationView
```

Comment:
nit: typically we have a line of whitespace above all comments

Reply:
fixed